code/src/components/Styles.scss

```diff
@@ -0,0 +1,185 @@
+/*** VARIABLES ***/
+
+$gray: #f2f0f0;
+$black: #000;
+$pink: #f4acac;
+$light-pink: #ffd5cd;
+$red: red;
+$green: green;
+
+/*** GENERAL ***/
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  font-family: menlo;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*** APP ***/
+
+.App {
+  margin: 50px auto;
+  width: 90%;
+  max-width: 500px;
+
+  &__grid {
+    display: grid;
+    row-gap: 50px;
+  }
+}
```

Comment:
The Scss looks interesting, but I am unfamiliar with it so I cannot really give you any feedback on it. But I think it's suits you using it to create structure and it does make the code easier to follow for me!